src/widgets/panel/utilitypanel.cpp

```diff
@@ -57,6 +57,7 @@ UtilityPanel::addToolWidget(QWidget* w)
   }
   if (w) {
     m_toolWidget = w;
+    m_toolWidget->setSizePolicy(QSizePolicy::Ignored, QSizePolicy::Preferred);
```

Comment:
Otherwise it tries too expand too much for some reason.

Not sure if it's the best solution, some parts of Qt layouting were always confusing to me, but everything else I tried did not work.

Reply:
I really like to understand why changes fix problems otherwise they always come back to haunt me :)

Reply:
Well, it means

> The sizeHint() is ignored. The widget will get as much space as possible.

which is probably what we want here because for some reason `sizeHint()` returns 391 width, which is bigger than the panel. 
Maybe `sizeHint` depends on the max item (font name) length in the combobox? 
But either way `Ignore` is probably a good way to fix because we don't want any tool widget to be bigger than the panel.